dev/scripts/denoBuildTransforms.ts

```diff
@@ -0,0 +1,58 @@
+import { fromPackageRoot, readJson } from "../runtime/fs.js"
+
+// eslint-disable-next-line max-lines-per-function
```

Comment:
I mean I do wish it was split up more but this is more work than I thought it would be and you did an impressive job 👍